src/linear_algebra/span.lean

```diff
@@ -186,9 +186,16 @@ by rw [submodule.span_union, p.span_eq]
 lemma span_sup : span R s ⊔ p = span R (s ∪ p) :=
 by rw [submodule.span_union, p.span_eq]
 
-lemma span_eq_supr_of_singleton_spans (s : set M) : span R s = ⨆ x ∈ s, span R {x} :=
+/- Note that the character `∙` U+2219 used below is different from the scalar multiplication
+character `•` U+2022. -/
+notation R`∙`:1000 x := span R (@singleton _ _ set.has_singleton x)
```

Comment:
Maybe this should be a localized notation? That dot looks like it could be useful for other things.

Reply:
I agree.

However note that I am not introducing his notation, I have just moved its definition c.120 lines higher in the file so that I can use it where I need it.

I contend that it might be better to localize the notation in a separate PR. It is already used in the following 29 files:
src/analysis/normed_space/hahn_banach/extension.lean
src/analysis/normed_space/operator_norm.lean
src/analysis/inner_product_space/pi_L2.lean
src/analysis/inner_product_space/basic.lean
src/analysis/inner_product_space/gram_schmidt_ortho.lean
src/analysis/inner_product_space/projection.lean
src/ring_theory/coprime/ideal.lean
src/ring_theory/noetherian.lean
src/geometry/euclidean/monge_point.lean
src/geometry/manifold/instances/sphere.lean
src/linear_algebra/bilinear_form.lean
src/linear_algebra/finite_dimensional.lean
src/linear_algebra/projective_space/basic.lean
src/linear_algebra/linear_pmap.lean
src/linear_algebra/affine_space/finite_dimensional.lean
src/linear_algebra/affine_space/affine_subspace.lean
src/linear_algebra/sesquilinear_form.lean
src/linear_algebra/span.lean
src/linear_algebra/quadratic_form/basic.lean
src/linear_algebra/dual.lean
src/linear_algebra/dimension.lean
src/linear_algebra/dfinsupp.lean
src/linear_algebra/linear_independent.lean
src/data/zmod/quotient.lean
src/algebra/module/torsion.lean
src/algebra/lie/engel.lean
src/algebra/lie/centralizer.lean
src/algebra/algebra/operations.lean
src/algebra/algebra/subalgebra/basic.lean